src/lib.rs

```diff
@@ -33,26 +33,25 @@ extern crate alloc;
 
 extern crate bitcoin_hashes;
 
-#[cfg(feature = "std")]
+#[cfg(any(feature = "alloc", feature = "std"))]
 extern crate unicode_normalization;
 
-#[cfg(feature = "rand_core")]
-pub extern crate rand_core;
 #[cfg(feature = "rand")]
 pub extern crate crate_rand as rand;
+#[cfg(feature = "rand_core")]
+pub extern crate rand_core;
 #[cfg(feature = "serde")]
```

Comment:
It would be way easier to review if there was a separate formatting patch up front.

Reply:
I'll force push again with formatting patch.

Reply:
c208e2e